start-a-local-cluster.md

```diff
@@ -21,7 +21,7 @@ Once you've [installed CockroachDB](install-cockroachdb.html), you can quickly s
     <button type="button" class="btn details collapsed" data-toggle="collapse" data-target="#details1">Details</button>
     <div id="details1" class="collapse">
       <ul>
-        <li> The <code>--insecure</code> flag sets client/server communication to insecure on the default port, 26257. To bind to different port, set <code>--port=&#60;port&#62;</code> and <code>--http-port=&#60;port&#62;</code>.</li>
+        <li> Client/server communication defaults to insecure, with the server listening only on <code>localhost</code> on port 26257. To bind to different port, set <code>--port=&#60;port&#62;</code> and <code>--http-port=&#60;port&#62;</code>.</li>
```

Comment:
You should mention that in order to listen (insecurely) on an external address the `--insecure` flag needs to be explicitly specified.